tests/integration/test_terraform.py

```diff
@@ -0,0 +1,181 @@
+"""Integration tests for the Terraform/Terragrunt module."""
+
+import json
+import logging
+import os
+import shutil
+import subprocess
+import tempfile
+import time
+from pathlib import Path
+from typing import Any
+
+import jubilant
+import pytest
+
+from tests.integration import helpers
+
+logger = logging.getLogger(__name__)
+pytestmark = pytest.mark.slow
```

Comment:
is this used anywhere ? (or is it a global config thing ?)

Reply:
This is indeed pytest-speak for "mark all tests in this module with the `slow` tag"